app/views/escorts/_risk.html.slim

```diff
@@ -1,12 +1,12 @@
 #risk
   = render partial: 'status',
-    locals: { workflow: risk,
-              not_started_path: new_escort_risks_path(escort),
-              summary_path: escort_risks_path(escort) }
+    locals: { workflow: risk, not_started_path: new_escort_risks_path(escort), summary_path: escort_risks_path(escort) }
 
   h2
     ' Risk
-    / small 16:44 | 04 Apr 2016
+    - if risk&.reviewer
```

Comment:
Where possible we should avoid using this methods in the controllers/views and let the object itself deal with it, having instead something like:

```ruby
risk.reviewed?
```